### DISCUSSION TITLE
Is it possible to schedule bat file execution in windows?

### DISCUSSION THREAD

**Discussion by andrec84j** [jhuckaby/Cronicle#518] [General]
I would like to know how to run bat files on remote windows machines on my network. Does anyone have any tips? Thanks.

**Comment by mikeTWC1984** [comment]
I guess most common way is to install openssh on your windows server and just run commands over ssh.
It is maintained by MSFT team:
https://github.com/PowerShell/Win32-OpenSSH
If you can get it to work I can share a plugin for invoking scripts via ssh.

Other option I guess is to use something like WinRM. Google for  python scripts using it. However it will likely require some configuration on windows end too.

**Comment by dheise-rheinenergie-trading-com** [comment]
We do it the same way like Mike. OpenSSH is a really uncomplicated and stable option to run any executable on a remote windows host.
